### ISSUE TITLE
Assess WCAG 2.1 Level AA compliance for state/local ADA rule

### ISSUE THREAD

**Issue by rcheetham** [opendataphilly/opendataphilly-jkan#578]
The City is subject to a new ADA rule requiring state and local governments to comply with web accessibility guidelines. 

https://www.ada.gov/resources/2024-03-08-web-rule/

As OpenDataPhilly does not have a contract with the City, I don't think we are legally required to comply, but I think we should still aim to assess our accessibility and outline areas in which we can improve.

The City is going to host an information session in December and I will try attend to learn more about their requirements. After that, I think we should test where we stand and then outline areas in which we might need to make improvements.
